docs/source/ops.rst

```diff
@@ -38,3 +44,12 @@ Tensor Contraction
     :member-order: bysource
 
 .. autofunction:: pyro.ops.contract.ubersum
+
+Statistical Utilities
+---------------------
+
+.. automodule:: pyro.ops.stats
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
```

Comment:
Docs for stats are missing. There aren't docs for many other ops too but it might be an intention so I don't touch them.